pythonforandroid/bootstraps/common/build/build.py

```diff
@@ -521,9 +533,18 @@ def make_package(args):
         versioned_name=versioned_name)
 
     # String resources:
+    timestamp = time.time()
+    if 'SOURCE_DATE_EPOCH' in environ:
+        # for reproducible builds
+        timestamp = int(environ['SOURCE_DATE_EPOCH'])
```

Comment:
How about using `get()` to make it slightly shorter?
```suggestion
    # for reproducible builds
    timestamp = int(environ.get('SOURCE_DATE_EPOCH', time.time()))
```
Note this is slightly different from your version above as in this version `time.time()` is always casted to `int`

Reply:
Normally I prefer shorter code :)
But in this case I think I prefer having the `if 'SOURCE_DATE_EPOCH' in environ:` as it makes the differences between reproducible and "normal" builds more obvious (and easy to grep).